plugins/postcss-cascade-layers/src/index.ts

```diff
@@ -22,7 +22,7 @@ const creator: PluginCreator<pluginOptions> = (opts?: pluginOptions) => {
 
 	return {
 		postcssPlugin: 'postcss-cascade-layers',
-		Once(root: Container, { result }: { result: Result }) {
+		OnceExit(root: Container, { result }: { result: Result }) {
```

Comment:
see : https://github.com/csstools/postcss-plugins/pull/374

This change will likely be merged and released before we can add it to `preset-env` so adding it here as it is an obvious improvement.